src/decomon/backward_layers/backward_layers.py

```diff
@@ -356,9 +355,11 @@ def get_config(self):
         return config
 
     def get_affine_components(self, x):
-        """Conv is a linear operator but its affine component is implicit
-           we use im2col and extract_patches to express the affine matrix
-           Note that this matrix is Toeplitz
+        """Express the implicit affine matrix of the convolution layer.
+
+        Conv is a linear operator but its affine component is implicit
+        we use im2col and extract_patches to express the affine matrix
+        Note that this matrix is Toeplitz
```

Comment:
It is okay now. Actually my comment was quite general as other functions in the new code was presenting the same kind of welll detailed but misformatted docstring. Maybe we could cope with all those docstrings in another PR because there is work throughout the library to make all docstrings be compliant with pep 257. Or you can make at least the new docstings respect it as you wish.